hydra-tui/src/Hydra/TUI.hs

```diff
@@ -172,22 +177,37 @@ warn :: Text -> State -> State
 warn = report Error
 
 report :: Severity -> Text -> State -> State
-report typ msg = feedbackL ?~ UserFeedback typ msg
+report typ msg st = case st of
+  Connected{me, nodeHost, peers, headState, dialogState, feedbacks, now} ->
+    Connected{me, nodeHost, peers, headState, dialogState, feedbacks = userFeedback : feedbacks, now}
```

Comment:
You could have used `~%` if I'm not mistaken. This is similar to `?~`, which is assigning, but `~%` is updating given a function (`userFeedback:` in our case)

Reply:
worked like a charm! 💝 
```
s & feedbackL %~ (userFeedback :)
```